scripts/benchmark.py

```diff
@@ -62,6 +62,7 @@ def main():
     args = p.parse_args()
 
     method = os.environ.get('METHOD', 'opt')
+    args.db = os.environ.get('MOOSE_SPEED_DB', args.db)
```

Comment:
Shouldn't command line win over environment variable?
The default for --db could just be `os.environ.get('MOOSE_SPEED_DB', "speedtests.sqlite")`

Reply:
done